### PR TITLE
Layout main #13

### PR DESCRIPTION
**Pull request by gosiamaks**
Add working navbar, app layout and WIP views for ingredients/index.
connected to  #13 connected to #37

### CODE REVIEW

File: app/assets/stylesheets/layout.scss.erb [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+.lunch-main-left {
+  margin: auto;
+  padding: 0;
+  text-align: center;
+}
+
+.lunch-main-left__box {
+  margin: 0;
+  padding: 40px;
+
```
Comment (basiacierocka) [inline app/assets/stylesheets/layout.scss.erb]: empty line

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -5,20 +5,29 @@
     <%= csrf_meta_tags %>
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```
Comment (basiacierocka) [inline app/views/layouts/application.html.erb]: too much space

**Comment by basiacierocka** [comment]
think its ok

**Comment by gotar** [comment]
ok you can merge

**Comment by gotar** [comment]
just squash and rebase with master

**Pull request merged**